File: domain/src/main/java/com/smartphoneShop/backend/dao/entity/Smartphones.java
package com.smartphoneShop.backend.dao.entity;


import com.sun.istack.NotNull;
import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

<<<<<<< HEAD
@Entity(name = "Smartphones")
=======
@Entity(name = "Smartphone")
>>>>>>> d81b6d3... Code review things
@Data
public class Smartphones {

    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    private int id;

    @NotNull
    private String brand;

    @NotNull
    private String model;

    @NotNull
    private int prize;

    @NotNull
    private int stock;
}

File: angularSmartphoneShop/src/app/application/domain/external/smartphones.ts

export class Smartphones {
  // yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
  constructor(public id?: number, public brand: string = '', public model: string = '',
    public prize: number = 0, public stock: number = 0) {
    this.id = id;
    this.brand = brand;
    this.model = model;
    this.prize = prize;
    this.stock = stock;

  }
}
